dev/provider_packages/prepare_provider_packages.py

```diff
@@ -766,40 +785,50 @@ def convert_pip_requirements_to_table(requirements: Iterable[str]) -> str:
         if found:
             package = found.group(1)
             version_required = found.group(2)
-            table_data.append((package, version_required))
+            if version_required == "":
+                version_required = ""
+            else:
```

Comment:
Can you explain this? Should `if version_required != ""` yield the same result?

Reply:
:D. Some left-over :). changed